spec/mailers/user_mailer_spec.rb

```diff
@@ -168,26 +167,25 @@
   end
 
   describe "Feedback invitation" do
-    let (:ac) { create(:associate_consultant) }
-    let (:review) { create(:review, associate_consultant: ac, feedback_deadline: Date.today) }
-    let (:email) { "recipient@example.com" }
-    let (:message) { "Hello. Please leave feedback." }
-    let (:mail) { UserMailer.review_invitation(review, email, message) }
-
-    it 'renders the subject' do
-      mail.subject.should == "[ReviewSite] You've been invited to give feedback for #{ac.user.name}"
-    end
-
-    it 'renders the receiver email' do
-      mail.to.should == ["recipient@example.com"]
-    end
-
-    it 'renders the sender email' do
-      mail.from.should == ['do-not-reply@thoughtworks.org']
-    end
-
-    it 'contains params[:message]' do
-      mail.body.encoded.should match("Hello. Please leave feedback.")
+    let(:ac)          { create(:associate_consultant) }
+    let(:review)      { build(:review, associate_consultant: ac, feedback_deadline: Date.today) }
+    let(:email)       { "recipient@example.com" }
+    let(:message)     { "Hello. Please leave feedback." }
+    let(:copy_sender) { false }
+    
+    subject { UserMailer.review_invitation(review, email, message, copy_sender) }
+    
+    its(:subject) { should == "[ReviewSite] You've been invited to give feedback for #{ac.user.name}" }
+    its(:to)      { should == [email] }
+    its(:from)    { should == ['do-not-reply@thoughtworks.org'] }
+    
+    it 'should have a body' do 
+      subject.body.encoded.should match("Hello. Please leave feedback.")
+    end
+
+    describe 'the sender is copied' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.